src/ValueParsers/CalendarModelParser.php

```diff
@@ -30,11 +30,12 @@ protected function stringParse( $value ) {
 		switch ( $value ) {
 			case '':
 			case 'gregorian':
-				return TimeFormatter::CALENDAR_GREGORIAN;
+				return IsoTimestampParser::CALENDAR_GREGORIAN;
```

Comment:
They don't "go" anywhere, they are already there, duplicated in both "formatter" and "parser" classes. I'm trying to deprecate at least one set of these constants now. This patch here is only a small step forward. More to come, as always.